src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -550,30 +550,26 @@ namespace
         }
     }
 
-    void write_png_image(
-        const bf::path&         file_path,
-        const Image&            image,
-        ImageAttributes         image_attributes)
+    void write_format_image(
+      const bf::path&         file_path,
+      const Image&            image,
+      ImageAttributes         image_attributes)
     {
-        Image transformed_image(image);
-
-        // todo: we may want to be more specific here.
-        if (image.properties().m_channel_count == 4)
-            transform_to_srgb(transformed_image);
-
-        create_parent_directories(file_path);
-
-        const std::string filename = file_path.string();
-        GenericImageFileWriter writer(filename.c_str());
-
-        writer.append_image(&transformed_image);
+      Image transformed_image(image);
+      // todo: we may want to be more specific here.
+      if (image.properties().m_channel_count == 4)
+          transform_to_srgb(transformed_image);
 
-        writer.set_image_output_format(PixelFormat::PixelFormatUInt8);
+      create_parent_directories(file_path);
 
-        image_attributes.insert("color_space", "sRGB");
-        writer.set_image_attributes(image_attributes);
+      const std::string filename = file_path.string();
+      GenericImageFileWriter writer(filename.c_str());
 
-        writer.write();
+      writer.append_image(&transformed_image);
+      writer.set_image_output_format(PixelFormat::PixelFormatUInt8);
```

Comment:
Just like @slongle PR, as far as I can tell this will break writing high dynamic range formats such as OpenEXR.